test/server_search.cpp

```diff
@@ -1456,6 +1456,32 @@ TEST(ServerSearchTest, searchResults)
         { "▶", 40, false },
       }
     },
+
+    {
+      /* query */          "pattern=j%C3%A0zz"
```

Comment:
Please add a comment explaining the purpose of this test point (so that maintainers don't have to resort to `git blame` to find it out).

Reply:
@mgautierfr Can we fix and merge please?

Reply:
Done (commit directly modified, no `fixup!`)